book/logging.md

```diff
@@ -0,0 +1,70 @@
+# RHTAP Logs
+
+Logging is crucial for effective monitoring and troubleshooting.\
+In the RHTAP environments, numerous clusters, components and services work together, making it essential to have comprehensive logs. Logging provides valuable insights into their behavior, allowing to understand and diagnose issues. Logging facilitates proactive monitoring, enabling prompt detection of failures, abnormal behavior, or potential security threats.\
+Additionally, logs play a vital role in compliance and auditing, helping to meet regulatory requirements and uncovering the root causes of incidents.\
+Ultimately, effective logging practices contribute to enhanced operational efficiency, streamlined troubleshooting, and maintaining a robust and secure environment.
+
+## Logs retention in Splunk
+
```

Comment:
I think adding here a one-liner about Splunk would help giving some context. `Splunk is a...`

Reply:
Good idea. Added.